src/analytics/Prices.js

```diff
@@ -0,0 +1,82 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+
+import { Col, Row } from 'reactstrap';
+
+import ErrorBoundary from 'components/ErrorBoundary';
+import { compareStrings, SortableTable } from 'components/SortableTable';
+import { cardEtchedPrice, cardFoilPrice, cardNormalPrice, cardPrice, cardStatus } from 'utils/Card';
+
+const Prices = ({ cards }) => {
+  const getValue = (collection, allowedStatus, allowedFinish) => {
+    const allowedStatuses = [];
+    if (allowedStatus === 'Owned' || allowedStatus === 'Any') allowedStatuses.push('Ordered', 'Owned', 'Premium Owned');
+    if (allowedStatus === 'Not Owned' || allowedStatus === 'Any') allowedStatuses.push('Not Owned', 'Proxied');
+
+    let priceFn;
+    switch (allowedFinish) {
+      case 'Non-foil':
+        priceFn = cardNormalPrice;
+        break;
+      case 'Foil':
+        priceFn = cardFoilPrice;
+        break;
+      case 'Etched':
+        priceFn = cardEtchedPrice;
+        break;
+      default:
+        priceFn = cardPrice;
+    }
+
+    return '$'.concat(
+      parseFloat(
+        collection
+          .map((card) => {
+            if (allowedStatuses.find((status) => status === cardStatus(card))) return priceFn(card) ?? 0;
+            return 0;
+          })
+          .reduce((acc, price) => acc + price),
+      ).toFixed(2),
+    );
+  };
+
+  const prices = useMemo(
+    () =>
+      ['Non-foil', 'Foil', 'Etched', 'All'].map((finish) => ({
+        label: finish,
```

Comment:
I feel like this labeling is a little unclear, in that it's not really self-evident what the cell values mean. For example, for `Foil -> To Complete`, does that mean the value of "Not Owned" cards that are marked as foil, the value of "Owned" cards that are not marked as foil but could be turned into foils, or the total value of all cards, regardless of status, that that aren't marked as foil? I can see justifications for any of the three.

Feel free to come up with your own way of clarifying this, but for my money, since the headings really just represent ownership status, why not label them that way? Meaning you'd just use "Owned" as a heading instead of "Current Value" etc. I feel like that's much more clear, if ever so slightly less natural.